### PR TITLE
Fixed header

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Fixed header. 

## Todo: 

- [ ] Rebase 

## Screenshots
capp agile learning defacto

### CODE REVIEW

File: source/stylesheets/_header.scss [hunk 1]
```diff
@@ -42,12 +45,12 @@
       }
 
       a {
-        @include transition(opacity 0.2s ease);
         color: $gray-menu;
         display: block;
         font-size: 1em;
         font-weight: bold;
         padding: 0 20px;
+        @include transition(opacity 0.2s ease);
```
Comment (RonaldDijkstra) [inline source/stylesheets/_header.scss]: d'oh

**Comment by RonaldDijkstra** [comment]
I tested this in IE10 and greater, Edge, Chrome, Firefox etc. All works fine afaik. 

Ready for review.

File: config.rb [hunk 2]
```diff
@@ -400,21 +400,19 @@ def capp_agile_url?
...
   def nav_white?
     current_page.data.nav_white
   end
 
+  def header_style
+    if current_page.data.header_style == "white"
+      "header-white"
+    elsif current_page.data.header_style == "invert"
+      "header-invert"
+    end
```
Comment (fatboypunk) [inline config.rb]:
```ruby
"header-#{current_page.data.header_style}"
```

File: source/localizable/_header.erb [hunk 3]
```diff
@@ -1,181 +1,190 @@
...
+            <li id="products" class="dropdown gallery">
+
+              <div class="content">
+                <h2 class="dropdown-title">
+                  <%= nav_link_to t("header.products"), "/producten.html" %>
+                </h2>
+
+                <ul>
+                  <% if root_locale == :nl %>
+                      <li>
```
Comment (fatboypunk) [inline source/localizable/_header.erb]: indentation

File: source/localizable/_header.erb [hunk 4]
```diff
@@ -1,181 +1,190 @@
...
                     <li>
-                      <% nav_link_to("/capp-entwickeln.html", { class: "nav-develop" }) do %>
-                        <h3><%= t("products.capp.develop") %></h3>
-                        <%= t("nav.capp-develop") %>
+                      <% nav_link_to("/capp-agile-learning.html", { class: "nav-capp-agile" }) do %>
+                        <h3><%= t("capp-agile.name") %></h3>
+                        <%= t("nav.capp-agile") %>
                       <% end %>
                     </li>
                   <% end %>
```
Comment (fatboypunk) [inline source/localizable/_header.erb]: We do a lot of if/elses on language, can't we just make 2 header files?

File: source/localizable/_header.erb [hunk 5]
```diff
@@ -1,181 +1,190 @@
...
+                  <%= nav_link_to t("header.services"), "/diensten.html" %>
+                </h2>
+
+                <ul class="links-list">
+                  <li><%= nav_link_to t("services.services.elearning.name"), "/e-learning.html" %></li>
+                  <li><%= nav_link_to t("services.services.implementaties.name"), "/implementaties.html" %></li>
+                  <% if root_locale == :de %>
+                    <li><%= nav_link_to t("services.services.hosting.name"), "/hosting.html" %></li>
+                  <% elsif root_locale == :nl %>
+                    <li><%= nav_link_to t("services.services.hosting.name"), "/hosting-security.html" %></li>
```
Comment (fatboypunk) [inline source/localizable/_header.erb]: So the url can be i18n'd right?

File: source/localizable/_header.erb (hunk 4 above)
Comment (RonaldDijkstra) [inline source/localizable/_header.erb]: Yes...

File: source/localizable/_header.erb (hunk 5 above)
Comment (sn3p) [inline source/localizable/_header.erb]: Middleman automatically translates URLs when provides in the translations.

File: source/stylesheets/_header.scss [hunk 6]
```diff
@@ -1,18 +1,105 @@
...
+    padding: 0;
+    position: fixed;
     text-align: right;
+    top: 0;
+    width: 100%;
+
+    &.header-fixed {
+      background: $white;
+      box-shadow: 0 4px 2px -2px rgba($black, 0.1);
+      transition: all 0.2s ease;
```
Comment (sn3p) [inline source/stylesheets/_header.scss]: Lets use specific transitions if possible, e.g.

```
transition: background 0.2s ease,
            box-shadow 0.2s ease;
```

And don't we need the transition on the default `.header`?

File: source/localizable/_header.erb (hunk 5 above)
Comment (fatboypunk) [inline source/localizable/_header.erb]: Yes so why do we need an if else?

Comment (sn3p) [inline source/localizable/_header.erb]: Because URLs are different in this case (not sure why).

Comment (RonaldDijkstra) [inline source/localizable/_header.erb]: Because the page in dutch isn't just about hosting, the path was changed last year.

Comment (fatboypunk) [inline source/localizable/_header.erb]: I thought they were translated, so that doesn't matter right?

Comment (fatboypunk) [inline source/localizable/_header.erb]: this will be resolved by https://github.com/DefactoSoftware/website/pull/585/files#r122745824 as well

Comment (RonaldDijkstra) [inline source/localizable/_header.erb]: I want to do this in another PR.

**Comment by RonaldDijkstra** [comment]
This is ready again, but will conflict with #593. 
Needs a rebase when that 1 gets merged.

File: source/stylesheets/_header.scss [hunk 7]
```diff
@@ -1,18 +1,105 @@
...
+        }
+
+        .word-mark {
+          fill: $gray-menu;
+        }
+      }
+
+      .main-nav {
+        &.header-white,
+        &.header-invert {
```
Comment (sn3p) [inline source/stylesheets/_header.scss]: `&.header {`

File: source/stylesheets/_header.scss [hunk 8]
```diff
@@ -1,18 +1,105 @@
...
+            &.active > a {
+              color: $orange;
+            }
+          }
+        }
+      }
+    }
+
+    &.header-invert,
+    &.header-white {
```
Comment (sn3p) [inline source/stylesheets/_header.scss]: `&.header {`

File: source/stylesheets/_header.scss (hunk 7 above)
Comment (RonaldDijkstra) [inline source/stylesheets/_header.scss]: Doesn't output the right styling here.

File: source/stylesheets/_header.scss (hunk 8 above)
Comment (RonaldDijkstra) [inline source/stylesheets/_header.scss]: Needed!
